packages/website/src/features/Packages/CodeExplorer.tsx

```diff
@@ -43,6 +98,41 @@ export const CodeExplorer: FC<{
     !!variant?.deploy_url
   );
 
+  // Provisioned packages could be inside the "provision" (old) or "clone" (current) key
+  // So we check both in order to keep backwards compatibility
+  const provisionedPackagesKeys =
+    deploymentData?.data?.def?.provision || deploymentData?.data?.def?.clone
+      ? Object.keys(
+          deploymentData?.data?.def?.provision ||
+            deploymentData?.data?.def?.clone
+        )
+      : [];
+  const provisionArtifacts = provisionedPackagesKeys.map((k: string) => {
+    return {
+      name: k,
+      artifacts: (
+        deploymentData?.data?.state[`provision.${k}`] ||
+        deploymentData?.data?.state[`clone.${k}`]
+      )?.artifacts,
+    };
+  });
```

Comment:
@mjlescano could you confirm this is the intended behavior?

Reply:
Yup, that looks right, so is still compatible with old packages and new ones, right?